docs/UserGuide.md

```diff
@@ -177,6 +177,20 @@ Format: `find_task KEYWORD [MORE_KEYWORDS]`
 Examples:
 * `find_task Book` returns `book` and `Find book`
 
+### Delete tasks:
+
+Deletes an existing task in task list.
+
+Format: `delete_task INDEX`
+
+* Deletes the task at the specified `INDEX`. The index refers to the index number shown in the displayed task list. The index **must be a positive integer** 1, 2, 3, …​
+* At least one of the optional fields must be provided.
```

Comment:
I think we can remove the optional fields here

Reply:
Resolved!